edge-auth/main.go

```diff
@@ -113,6 +113,7 @@ func main() {
 		"/function/audit-event",
 		"/function/echo",
 		"/function/metrics",
+		"/function/function-logs",
```

Comment:
Did you rename the pipeline logs too?

Reply:
Not in the backend - think "pipeline-log" is a good name for backend function